examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<ol><li>First, add the description of what <code>HierarchicalStructure</code> is, sth like "To handle the hierarchical structure of the data we need to create a special object ...."</li><li>Then, describe how to create it: "To define the <code>HierarchicalStructure</code>  you need to pass the adjacency list setting the connections between segments and list with names of the levels in the hierarchy from top to bottom"</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>